voter/constants.py

```diff
@@ -209,3 +209,8 @@
     ('WI', 'Wisconsin'),
     ('WY', 'Wyoming'),
 ]
+
+STATE_FILTER_CHOICES = [
+    (str(abb), name, "were born in <em>%s</em>" % name)
```

Comment:
I don't think we need the `str()` in here. We needed it for the counties because the ID was an integer.